cmd/podman/containers/ps.go

```diff
@@ -142,11 +142,19 @@ func checkFlags(c *cobra.Command) error {
 }
 
 func jsonOut(responses []entities.ListContainer) error {
-	r := make([]entities.ListContainer, 0)
+	type jsonFormat struct {
+		entities.ListContainer
+		Created int64
+	}
+	r := make([]jsonFormat, 0)
 	for _, con := range responses {
 		con.CreatedAt = units.HumanDuration(time.Since(con.Created)) + " ago"
 		con.Status = psReporter{con}.Status()
-		r = append(r, con)
+		jf := jsonFormat{
+			ListContainer: con,
+			Created:       con.Created.UnixNano(),
```

Comment:
This doesn't look correct. Earlier it used the be the UNIX timestamp in seconds.

Like I mentioned in https://github.com/containers/podman/issues/9315, Toolbox no longer uses the `CreatedAt` field for containers, because its different for images anyway. So maybe `CreatedAt` could be repurposed for a more precise time representation? That could possibly make it consistent with images.

Reply:
Or, is there a way to programmatically differentiate between a timestamp in seconds and one in nanoseconds? Then Toolbox could handle that while parsing the JSON value.